include/swift/Frontend/FrontendOptions.h

```diff
@@ -139,6 +139,12 @@ class FrontendOptions {
     /// Parse, type-check, and dump type refinement context hierarchy
     DumpTypeRefinementContexts,
 
+    /// Parse, type-check, and dump a list of Unified Symbol Resolutions
+    /// (USRs)for each function declaration that matches what XCTest considers
+    /// to be a viable test: a class instance method that takes no parameters,
+    /// returns void, and begins with "test".
+    DumpXCTestMethods,
```

Comment:
I wonder... instead of being an ActionType, which implies that it cannot be combined with (say) another compiler. I guess it doesn't matter much for now, but it seems like SourceKit would want to be able to answer this query for an already-parsed source file, and not have to do another driver invocation. Maybe that means going through the API that's currently on ValueDecl.